### PR TITLE
fix: fix alignment issues in onboarding success

### LINKED ISSUE #9879: [Bug]: "Done" Button on "Confirm Secret Recovery Phrase" screen covers up some text
### Describe the bug

On the "Confirm Secret Recovery Phrase" screen the blue "Done" button covers up the bottom line of text.

Device info:
image

### Expected behavior

Perhaps it should be a scrollview if there isn't enough space for all of the text

### Screenshots/Recordings

image

### Steps to reproduce

1. Start up app with fresh installation
2. Go through wallet creation process including secret recovery phrase

### Error messages or log output

```shell
n/a
```

### Version

7.23.0

### Build type

Other (please specify exactly where you obtained this build in "Additional Context" section)

### Device

Android (see earlier screenshot)

### Operating system

Android

### Additional context

Commit: 2384af6f675f82d3abbf13f3b31f6095922c0ac7

### Severity

Not severe

### PR DESCRIPTION
**Pull request by brianacnguyen**
## **Description**
- This PR fixes the alignment issues in onboarding success

## **Related issues**

Fixes: #9879 

## **Manual testing steps**

1. Go to onboarding process
2. Stop once you're done with the secret recovery process
3.

### **Before**
Simulator Screenshot - iPhone 15 Pro - 2024-07-21 at 20 24 35
Simulator Screenshot - iPhone SE (3rd generation) - 2024-07-21 at 20 06 18

### **After**
Simulator Screenshot - iPhone 15 Pro - 2024-07-21 at 20 29 58
Simulator Screenshot - iPhone SE (3rd generation) - 2024-07-21 at 20 38 24

## **Pre-merge author checklist**

- [x] I’ve followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Mobile Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Review by georgewrmarshall — COMMENTED** [review]
Looking good! Looks like it needs a snapshot test update

File: app/components/Views/OnboardingSuccess/index.styles.ts [hunk 1]
```diff
@@ -3,15 +3,15 @@ import { StyleSheet } from 'react-native';
...
   contentWrapper: {
     flex: 1,
-    paddingTop: '10%',
+    paddingTop: 24,
+    padding: 24,
   },
   buttonWrapper: {
-    width: '85%',
-    bottom: 50,
+    paddingHorizontal: 24,
```
Comment (georgewrmarshall) [inline app/components/Views/OnboardingSuccess/index.styles.ts:14]: Should we add some additional bottom padding here so there is some white space between the button and screen bottom

Screenshot 2024-08-12 at 11 35 28 AM

**Review by georgewrmarshall — DISMISSED** [review]
LGTM! Left one non-blocking comment
- tested locally on small screen iphone SE (3rd generation) ✅

File: app/components/Views/OnboardingSuccess/index.styles.ts (hunk 1 above)
Comment (brianacnguyen) [reply app/components/Views/OnboardingSuccess/index.styles.ts:13]: Updated
Simulator Screenshot - iPhone SE (3rd generation) - 2024-08-12 at 12 46 28

**Review by georgewrmarshall — APPROVED** [review]
LGTM!

**Pull request merged**